backend/papersfeed/tests/tests_user.py

```diff
@@ -40,6 +40,35 @@ def setUp(self):
                     }),
                     content_type='application/json')
 
+    def test_csrf(self):
+        """ CSRF TOKEN TEST """
+        # By default, csrf checks are disabled in test client
+        # To test csrf protection we enforce csrf checks here
+        client = Client(enforce_csrf_checks=True)
+        response = client.post('/api/user',
+                               json.dumps({
+                                   constants.EMAIL: 'csrf@snu.ac.kr',
+                                   constants.USERNAME: 'scrf',
+                                   constants.PASSWORD: 'iluvswpp1234'
+                               }),
+                               content_type='application/json')
+        self.assertEqual(response.status_code, 403)  # Request without csrf token returns 403 response
+
+        response = client.get('/api/token')
+
+        csrf_token = response.cookies['csrftoken'].value  # Get csrf token from cookie
+
+        response = client.post('/api/user',
+                               json.dumps({
+                                   constants.EMAIL: 'csrf@snu.ac.kr',
+                                   constants.USERNAME: 'csrf',
+                                   constants.PASSWORD: 'iluvswpp1234'
+                               }),
+                               content_type='application/json',
+                               HTTP_X_CSRFTOKEN=csrf_token)
+
+        self.assertEqual(response.status_code, 201)  # Pass csrf protection
```

Comment:
Actually, this line is the only difference from #84 PR.

Reply:
Because #75 PR changed the status code of POST APIs as 201, the assumption of #84 PR became wrong. That made this test fail on the master branch after merged.